evalJsonWithCallback({0}, getFunctionBody(() => {{
        try {{
           {1}
        }}
        catch (x) {{
            _LOG("Call Exception: " + x);
        }}
}}));